microsoft/botframework-solutions
# [Calendar] Fix some bugs

## Description

1. Fix bug that state is cleared when call update or change event status in summary flow. Add clear function for this case, and add NewStartDate, NewStartTime, NewEndDate, NewEndTime in state to avoid StartDate, StartTime, EndDate and EndTime are reused in update flow.
2. Change the max line of meeting content in the card to 3.
3. Add the cross flow test case.

## Related Issue





## Testing Steps


Try to update the meeting in summary flow. say "update the meeting" after bot asked "Would you like to see your calendar overview again for tomorrow? You can also ask me to update or decline this event for you."

## Checklist

If this contains changes that needs to be replicated between the Enterprise Template <-> Virtual Assistant

If you have updated responses or `.lu` files:
